pkg/kn/commands/plugin/plugin_verifier.go

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// +build !windows
```

Comment:
builds for non-Windows platform

Reply:
So, when we mention `GOOS=windows` while building cross platform binary it skips this file and picks up Windows specific version defining the needed methods / functions to verify plugin on Windows platform.